wrench/src/json_frame_writer.rs

```diff
@@ -1,21 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 // the json code is largely unfinished; allow these to silence a bunch of warnings
 #![allow(unused_variables)]
 #![allow(dead_code)]
 
 use image::{ColorType, save_buffer};
+use serde_json;
 use std::borrow::BorrowMut;
 use std::collections::HashMap;
-use std::fmt;
-use std::fs;
 use std::fs::File;
 use std::io::Write;
 use std::path::{Path, PathBuf};
+use std::{fmt, fs};
+use super::CURRENT_FRAME_NUMBER;
```

Comment:
it seems slightly wrong to me that `super::*` includes are mixed with the others